BTT/BeyondTheTutor/BeyondTheTutor/Models/Tutor.cs

```diff
@@ -18,7 +24,7 @@ public partial class Tutor
         [StringLength(50)]
         public string LastName { get; set; }
 
-        public DateTime ClassOf { get; set; }
+        public short ClassOf { get; set; }
```

Comment:
Please don't change this... make it into int16 instead of "short" theyre the same thing but i have int16 in like 6 locations :(